### PR TITLE
[ZH] Update Main_page

### PR DESCRIPTION
**Pull request by heipizhu4**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by kj415j45 — APPROVED** [review]

**Comment by kj415j45** [comment]
somehow duplicated with #8206, take care of potential conflict!

File: wiki/Main_Page/zh.md [hunk 1]
```diff
@@ -97,10 +95,10 @@ osu! 的谱面都是充满活力的社区内玩家们的智慧结晶，想与大
 # 社区与公众项目
 
 社交能把世界各地的 osu! 玩家连结起来。了解这些活动与竞赛，结识新朋友，或帮忙组织一个更好的活动！
-
+场所: [论坛](/wiki/Community/Forum) • [IRC](/wiki/Community/Internet_Relay_Chat)
```
Comment (agatemosu) [inline wiki/Main_Page/zh.md:98]:
```suggestion

场所: [论坛](/wiki/Community/Forum) • [IRC](/wiki/Community/Internet_Relay_Chat)
```

**Comment by YumeMuzi** [comment]
> somehow duplicated with #8206, take care of potential conflict!

It's a complete coincidence lol

**Comment by agatemosu** [comment]
Your PR has two corrections on lines 62 and 64, this PR does not have them.

**Comment by heipizhu4** [comment]
> #8206 has two corrections on lines 62 and 64, this PR does not have them.

Resolved

**Review by TicClick — APPROVED** [review]

**Pull request merged**